statemachine/state.py

```diff
@@ -208,6 +213,59 @@ def initial(self):
     def final(self):
         return self._final
 
+
+class InstanceState(State):
```

Comment:
A thin "proxy" for `State` that has `is_active` with a reference to a concrete instance of the SM.